Readme.md

```diff
@@ -16,42 +16,89 @@ Async-Transformers is a tiny no-frills no-dependencies ts-first implementation o
 
 ![Overview of async transformer functionality](./assets/async-transformers.png)
 
-The method `asyncBufferedTransformer()` was inspired by [`rust futures buffered()`](https://docs.rs/futures/latest/futures/stream/trait.StreamExt.html#method.buffered).
+The method `asyncBufferedTransformer()` was inspired by [rust futures `buffered()`](https://docs.rs/futures/latest/futures/stream/trait.StreamExt.html#method.buffered).
 
-## Usage
+## Example
+
+Here's an example that downloads all status code images from [http.cat](https://http.cat), but only 7 at a time to be a good
+internet citizen.
+
+You can run this using `npx ts-node examples/fetch-http-cats.ts`
+
+(Note: this library does _not_ depend on nodejs (and has zero dependencies), just this example)
```

Comment:
```suggestion
(Note: this library does _not_ depend on node-fetch (and has zero dependencies), just this example does)
```

Reply:
thats what you meant, right?

(i guess it could be both, but it was confusing to me either way so maybe rephrase it either way)